CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.15 (TBD)
 
+### etcd server
+- Fix [add prometheus metric registration for metric `etcd_disk_wal_write_duration_seconds`](https://github.com/etcd-io/etcd/pull/18164).
```

Comment:
Link should be pointing to the `release-3.5` backport pr: https://github.com/etcd-io/etcd/pull/18174

Reply:
> Link should be pointing to the `release-3.5` backport pr: #18174

@jmhbnz Done